### ISSUE TITLE
Need a 3.0 bugfix release for lumen, In Release 3.0.1 LumenServiceProvider's getRouter() function returns app not router

### ISSUE THREAD

**Issue by KealJones** [barryvdh/laravel-debugbar#727]
This has actually already been fixed in the master branch (#718) but there hasn't been a Release to fix this issue for lumen users yet. Can you release a bugfix version for us?

If you're having this issue and looking for a temporary fix, i have a temporary way around this that *doesn't* involve changing the vendor files. This may be obvious but I wanted to share in case anyone doesn't know how to go about it.

Create a file `DebugbarServiceProvider.php` in `app/Providers` with this content:
```
<?php namespace App\Providers;

use Laravel\Lumen\Application;
use Barryvdh\Debugbar\LumenServiceProvider;

class DebugbarServiceProvider extends LumenServiceProvider
{
    /** @var  Application */
    protected $app;

    /**
     * This is a temporary fix for lumen 5.5 bug in Barryvdh\Debugbar
     *
     * @return Application
     */
    protected function getRouter()
    {
        /**
         * In Barryvdh\Debugbar\LumenServiceProvider it is defined as:
         * return $this->app;
         * 
         * Fixed in: https://github.com/barryvdh/laravel-debugbar/pull/718
         */
        return $this->app->router;
    }
}

```

Then in `bootstrap/app.php` for now comment out the official service provider and add yours:
```
//$app->register(Barryvdh\Debugbar\LumenServiceProvider::class);
$app->register(App\Providers\DebugbarServiceProvider::class);
```

**Comment by barryvdh** [comment]
Released 3.1.0 which includes it.

**Comment by KealJones** [comment]
Thank you very much!
